web/src/main/webapp/js/addToCart.js

```diff
@@ -9,18 +9,22 @@ function addToCart(id) {
                 'Content-Type': 'application/json'
             },
             type: "POST",
-            url: "ajaxCart/",
+            url: "http://localhost:8080/phoneshop-web/ajaxCart/",
```

Comment:
Try to make your code environment agnostic. Say if you will need to publish that code in production environment, that won't work